cli/js/lib.deno.ns.d.ts

```diff
@@ -1478,7 +1478,7 @@ declare namespace Deno {
    * points to.
    *
    *       const fileInfo = await Deno.lstat("hello.txt");
-   *       assert(fileInfo.isFile);
+   *       console.assert(fileInfo.isFile);
```

Comment:
The `assert()` here refers to the one in our `std` assertion lib (or any assertion lib really), even if no import is shown.